pkg/windowsdriver/procmon/procmon.go

```diff
@@ -73,7 +79,13 @@ func NewWinProcMon(onStart chan *ProcessStartNotification, onStop chan *ProcessS
 	if err := driver.StartDriverService(driverName); err != nil {
 		return nil, err
 	}
-	reader, err := olreader.NewOverlappedReader(wp, procmonReceiveSize, procmonNumBufs)
+	if bufsize == 0 {
+		bufsize = procmonReceiveSize
+	}
+	if numbufs == 0 {
+		numbufs = procmonNumBufs
+	}
```

Comment:
suggestion: the `== 0` checks can be removed if we pass in the defaults when the function is called (see other comment about exporting them).